### PR TITLE
remove nightly ray, and use newest release version

### PR DESCRIPTION
**Pull request by harborn**

### CODE REVIEW

File: common/trainer/default_trainer.py [hunk 1]
```diff
@@ -57,8 +57,10 @@ def recovery(self, config):
                 self.starting_epoch = checkpoint_epoch["epoch"] + 1
 
             logger.info(f"recovery to epoch {self.starting_epoch}")
+        except FileNotFoundError as e:
+            logger.info(e)
```
Comment (jiafuzha) [inline common/trainer/default_trainer.py:61]: is it correct?

File: finetune/finetune.py [hunk 2]
```diff
@@ -108,6 +108,7 @@ def train_func(config: Dict[str, Any]):
     trainer = common.trainer.Trainer.registory.get("DefaultTrainer")(config = {
         "num_train_epochs": config["Training"]["epochs"],
         "max_train_step": config["Training"].get("max_train_steps", None),
+        "log_step": 1,
```
Comment (jiafuzha) [inline finetune/finetune.py:111]: what's the default value? Will it cause too much log output?

Comment (harborn) [reply finetune/finetune.py:111]: default value is 1

File: common/trainer/default_trainer.py (hunk 1 above)
Comment (harborn) [reply common/trainer/default_trainer.py:61]: Currently will print a ValueError if no checkpoint file found, this update will change ValueError to a normal info log

**Pull request merged**